src/ScriptCs.Core/ScriptLibraryWrapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ScriptCs.Contracts;
+
+namespace ScriptCs
+{
+    public abstract class ScriptLibraryWrapper
```

Comment:
this class does not surface `Env` - I know it's not on `IScriptHost` but it might be a good moment to ask if it should be?